registrations/filters.py

```diff
@@ -62,6 +62,13 @@ def filter_queryset(self, queryset):
             Q(registration__event__publisher__in=self._user_admin_organizations)
             & Q(registration__created_by=user)
         )
+
+        if user.email and user.email.endswith("@hel.fi"):
```

Comment:
Maybe required domain ("hel.fi") could be defined in the environment variables. That would make local testing and development easier. At least I don't have hel.fi email account any more.

Reply:
I added an environment variable and a setting that accepts a list as the value so multiple accepted domains are now supported. Default element in the list will be "@hel.fi". There is also a shared util function now that checks a given email against the acceptable domains (`has_allowed_substitute_user_email_domain`).